internal/install/execution/platform_link_generator.go

```diff
@@ -52,7 +52,11 @@ func (g *PlatformLinkGenerator) GenerateLoggingLink(entityGUID string) string {
 // also provided in the nerdstorage document. This provides the user two options
 // to see their data - click from the CLI output or from the frontend.
 func (g *PlatformLinkGenerator) GenerateRedirectURL(status InstallStatus) string {
-	if status.AllSelectedRecipesInstalled() {
+	installedCount := len(status.Installed)
+	haveFailure := status.HasUnsupportedRecipes && status.HasFailedRecipes
```

Comment:
Just double checking, should this use the `||` operator?

Reply:
Thanks, Sander, that was a good catch.  I had the wrong logic on the test builder, hence the test was passing.  Should be good now.